tests/test_core.py

```diff
@@ -436,3 +436,47 @@ def get_geometry(partition):
     result = dask_obj.map_partitions(get_geometry).compute()
     expected = dask_obj.geometry.compute()
     assert_geoseries_equal(result, expected)
+
+
+class TestDissolve:
+    def setup_method(self):
+        self.world = geopandas.read_file(
+            geopandas.datasets.get_path("naturalearth_lowres")
+        )
+        self.ddf = dask_geopandas.from_geopandas(self.world, npartitions=4)
+
+    def test_default(self):
+        gpd_default = self.world.dissolve("continent")
+        dd_default = self.ddf.dissolve("continent").compute()
+        assert_geodataframe_equal(gpd_default, dd_default, check_like=True)
+
+    def test_sum(self):
+        gpd_sum = self.world.dissolve("continent", aggfunc="sum")
+        dd_sum = self.ddf.dissolve("continent", aggfunc="sum").compute()
```

Comment:
The fact that this works (and automatically drops columns) seems a bit at odds with the note you added in the docstring?

Reply:
It doesn't drop columns, I drop them manually on line 458 before the check. It does a sum of strings so you get things like `RussiaNorwayFranceSwedenBelarusUkrainePolandAu...`.

So it is in line with the docstring above.

Reply:
Ah, yes, sum "works" for string columns ..